2016/playbook/session_2/SCHEDULE.md

```diff
@@ -0,0 +1,29 @@
+**Table of Contents**
+
+- [Days](#days)
+  - [Day 1](#day-1)
+
+# Days
+
+## Day 1
+
+| Time     | Activity                                                      |
+| -------- | ----------------------------------------------------------    |
+| 8:30 AM  | Standups                                                      |
+| 9:00 AM  | Hacker arrival                                                |
+| 9:10 AM  | Name Game                                                     | 
+| 9:20 AM  | Building preview                                              |
+| 9:25 AM  | [Energizer 1](../ACTIVITIES.md#i-love-my-neighbor-who)        |
+| 9:35 AM  | [Energizer 2](../ACTIVITIES.md#human-pictionary)              |
+| 9:45 AM  | Breakfast                                                     |
+| 10:15 AM | [Introduction talk](../ACTIVITIES.md#introduction-talk)       |
+| 10:25 AM | [Personal Website][personal_website]                          |
+| 12:30 PM | Lunch                                                         |
+| 1:30 PM  | [Energizer 3](../ACTIVITIES.md#evolution-rock-paper-scissors) |
+| 2:00 PM  | [Personal Website (continued)][personal_website]              |
+| 4:40 PM  | Collect links for demos                                       |
+| 5:00 PM  | [Demos](../ACTIVITIES.md#demos)                               |
+| 5:25 PM  | [Feedback form](../ACTIVITIES.md#feedback-forms)              |
```

Comment:
5:25, yeah. Should I change that to "Feedback sent by Howdy"?

On Mon, Jun 20, 2016 at 2:33 PM Max Wofford <notifications@github.com>
wrote:

> In 2016/playbook/session_2/SCHEDULE.md
> <https://github.com/hackclub/hackcamp/pull/694#discussion_r67770834>:
>
> > +| -------- | ----------------------------------------------------------    |
> > +| 8:30 AM  | Standups                                                      |
> > +| 9:00 AM  | Hacker arrival                                                |
> > +| 9:10 AM  | Name Game                                                     |
> > +| 9:20 AM  | Building preview                                              |
> > +| 9:25 AM  | [Energizer 1](../ACTIVITIES.md#i-love-my-neighbor-who)        |
> > +| 9:35 AM  | [Energizer 2](../ACTIVITIES.md#human-pictionary)              |
> > +| 9:45 AM  | Breakfast                                                     |
> > +| 10:15 AM | [Introduction talk](../ACTIVITIES.md#introduction-talk)       |
> > +| 10:25 AM | [Personal Website][personal_website]                          |
> > +| 12:30 PM | Lunch                                                         |
> > +| 1:30 PM  | [Energizer 3](../ACTIVITIES.md#evolution-rock-paper-scissors) |
> > +| 2:00 PM  | [Personal Website (continued)][personal_website]              |
> > +| 4:40 PM  | Collect links for demos                                       |
> > +| 5:00 PM  | [Demos](../ACTIVITIES.md#demos)                               |
> > +| 5:25 PM  | [Feedback form](../ACTIVITIES.md#feedback-forms)              |
>
> Is this when Howdy is scheduled to send out feedback forms?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/hackclub/hackcamp/pull/694/files/212fece1a32964c67aea120a134f6b2a381830a0#r67770834>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHYhMoDDod4uCxUEc8SjAUjXXsZ7lEojks5qNwcigaJpZM4I6Edx>
> .
>